### PR TITLE
Make entry page responsive using Bootstrap

### PR DESCRIPTION
**Pull request by kots14**
Adding responsive home page with required styles along with bootstrap 4. Reference - https://github.com/KamandPrompt/baat-cheet/pull/84

**Issue :** #66
**Reference conversation:** #70 (comment)
**Summary of changes:**
Updated all files in `master` branch from files present in `70-merged` branch.

**Testing:**
- [x] Tested responsiveness in Google Chrome
- [x] Tested responsiveness in Mozilla Firefox

### CODE REVIEW

**Comment by kots14** [comment]
@LalitNM please let me know if this needs any other change. I've checked the git logs and there's only one commit now where I've included all the files to avoid the mess. Thanks for your guidance in this regard :)

File: public/style.css [hunk 1]
```diff
@@ -648,10 +654,6 @@ body {
   background: #f8f8f8;
 }
 
-#room input, #room textarea {
-  font-size:1.1em;
-}
-
```
Comment (LalitNM) [inline public/style.css]: You removed this which reverts [this](https://github.com/KamandPrompt/baat-cheet/commit/dcb5027d468cbfe5e50267428e1ff30abfa244c1).

**Review by LalitNM — CHANGES_REQUESTED** [review]
Make required changes and again clean git history.

File: public/style.css (hunk 1 above)
Comment (kots14) [reply public/style.css]: I fixed this and did a `squash`. Let me know if it helps or not.

**Comment by kots14** [comment]
closing this PR as git logs are not fixed. I'll be adding a new PR.

**Pull request closed without merging**